src/abstractClass.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Vendor\Hw3;
+
+abstract class abstractClass
```

Comment:
Class names should be CamelCase (in your case first letter should be uppercase: AbstractClass )